### PR TITLE
Adding Acateo AI and minor fixes

### PR DESCRIPTION
**Pull request by Gladicek**
Nothing epic, skills/items are not commited cause of conflicts after some changes...

### CODE REVIEW

File: L2J_DataPack/dist/game/data/scripts/ai/npc/Acateo/Acateo.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		String htmltext = null;
+		switch (event)
+		{
+			case "give_circlet":
+			{
+				if (hasQuestItems(player, ACADEMY_CIRCLET))
+				{
+					htmltext = "33905-3.html";
+				}
+				giveItems(player, ACADEMY_CIRCLET, 1);
```
Comment (jurchiks) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/Acateo/Acateo.java]: this should be in an ELSE block...

Comment (St3eT) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/Acateo/Acateo.java]: There should not be switch at all.

Comment (Gladicek) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/Acateo/Acateo.java]: Totally pointless to care if there should be switch or classic event.equal...still its gonna work same way.

Comment (St3eT) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/Acateo/Acateo.java]: Equals is faster in this case.

Comment (UnAfraid) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/Acateo/Acateo.java]: missing break :)

**Comment by Sdw-** [comment]
That one ? https://github.com/L2J/L2J_DataPack/pull/69#discussion_r22770303 , lol ...

**Comment by jurchiks** [comment]
Ahh, "outdated diff"... That's stupid.
Nevertheless, no "Reported by: ".

**Pull request merged**